src/FSharp.Editing.VisualStudio/Coloring/UnusedDeclarationMargin.fs

```diff
@@ -1,4 +1,4 @@
-﻿namespace FSharpVSPowerTools
+﻿namespace FSharp.Editing.VisualStudio.Coloring.UnusedSymbols
```

Comment:
Why sub-namespace UunsedSymbols? Could we remove that for consistency with others?